irods/files/rule-bases/cyverse_logic.re

```diff
@@ -498,6 +498,7 @@ _cyverse_logic_sendEntityRm(*Type, *Id, *Path, *AuthorName, *AuthorZone) {
 
 # Indicates whether or not an AVU is protected
 _cyverse_logic_isAVUProtected(*Attr) = cyverse_startsWith(*Attr, 'ipc')
+_cyverse_logic_isAVUProtectedUUID(*Attr) = *Attr == _cyverse_logic_UUID_ATTR
```

Comment:
Separate these functions with a blank link to make it clear that they are separate functions.